gateway-ha/pom.xml

```diff
@@ -140,6 +140,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <scope>test</scope>
+        </dependency>
+
         <dependency>
```

Comment:
@mosabua It will throw testng exception if we remove it at this time. I think it is best to remove testng from pom once we remove all imports. We still use testng in baseapp module, and I'll fix it after this PR

Reply:
No .. lets clean it up in this module completely. Remove all imports in this module and if necessary override the surefire config. Lets clean it up for real... that way nobody can accidentally add a testng test in again

Reply:
ok fixed